python/lsst/cp/pipe/deferredCharge.py

```diff
@@ -540,7 +559,9 @@ def findTraps(self, inputMeasurements, calib, detector):
                     new_signal.append(exposureDict[ampName]['LAST_MEAN'])
                 else:
                     Nskipped += 1
-            self.log.info(f"Skipped {Nskipped} exposures brighter than {self.config.maxSignalForCti}.")
+            self.log.info(f"Skipped {Nskipped} exposures brighter than {self.config.maxImageMean}.")
```

Comment:
The additions look good (the new log messages), but I'm wondering if this code where you added the log message is repeated in all these functions (starting at the line with the comment that says "The signal is the mean intensity...."). Is this OK? Should it be refactored to avoid code duplication? (for another ticket, maybe)

Reply:
I wanted to keep the differences from the original code to a minimum, at least until it's been thoroughly tested.  I'm going to leave it for now, and will think about it in the future.